terraform/modules/api-gateway/api_gateway_lambda.js

```diff
@@ -1,11 +1,69 @@
+const AWS = require('aws-sdk');
```

Comment:
Not that it makes much difference, but we've tended to use the lower level clients for this sort of thing. See the other lambas in the src/ dir

Reply:
I’ve replaced the manual Lambda definition with the Terraform Lambda module, following the same structure as the ingestion Lambda

Reply:
I’ve updated `api_gateway_lambda.js` to use @aws-sdk/client-secrets-manager instead of aws-sdk, which keeps it consistent with other Lambdas and improves performance